examples/rlhf/requirements.txt

```
@@ -8,4 +8,4 @@ tiktoken
 tqdm
 transformers
 git+https://github.com/pytorch/rl
-git+https://github.com/pytorch-labs/tensordict
+git+https://github.com/meta-pytorch/tensordict
```

```suggestion
git+https://github.com/pytorch/tensordict
```
@ZainRizvi probably a remnant of a past life - if tensordict is not moving to `meta-pytorch` we can simply use